edge-apps/countdown-timer/screenly.yml

```diff
@@ -0,0 +1,44 @@
+---
+user_version: v1.0.0
+description: Screenly Countdown Edge App
+icon: https://playground.srly.io/edge-apps/clock/static/images/icon.svg
+author: Screenly, Inc.
+entrypoint: index.html
+settings:
+  enable_analytics:
+    type: string
+    default_value: 'true'
+    title: enable_analytics
```

Comment:
Should be `Enable Analytics` (and fix this for all titles in the file).

Reply:
Changed all the titles as per suggestion.